### PR TITLE
HPCC-21950 Document Logical File Copy Expire in (days) option

### PR DESCRIPTION
**Pull request by g-pan**
Signed-off-by: G-Pan <greg.panagiotatos@lexisnexis.com>

@JamesDeFabia please review
@miguelvazq please review (optional) 
@kunalaswani optionally review

Please Review compiled documentation http://10.240.32.243/view/Docs%207.0/job/DocDevII/
See the ECL Watch Manual (Find: Expire in (days))

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [x] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [x] My change requires a change to the documentation.
  - [x] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [x] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

## Testing:

Successful Build tests
http://10.240.32.243/view/Docs%207.0/job/DocDevII/

### CODE REVIEW

File: docs/EN_US/ECLWatch/ECLWa_mods/ECLWatchSrc.xml [hunk 1]
```diff
@@ -1387,6 +1387,55 @@
           <para>Press the <emphasis role="bold">Copy</emphasis> button to
           display the copy drop menu, with additional file copy
           options.</para>
+
+          <!--Now this here would be a really good spot to XInclude a dynamically built options table-->
```
Comment (JamesDeFabia) [inline docs/EN_US/ECLWatch/ECLWa_mods/ECLWatchSrc.xml]: remove this comment, if you want to do this, create a Jira

File: docs/EN_US/ECLWatch/ECLWa_mods/ECLWatchSrc.xml [hunk 2]
```diff
@@ -1387,6 +1387,55 @@
...
+              <listitem>
+                <para>Check the<emphasis role="bold"> Preserve
+                Compression</emphasis> box to retain the file
+                compression.</para>
+              </listitem>
+
+              <listitem>
+                <para>Check the<emphasis role="bold"> Expire in
+                (days)</emphasis> box to enter a number of days before
+                automatically removing the file from the queue. If omitted,
```
Comment (JamesDeFabia) [inline docs/EN_US/ECLWatch/ECLWa_mods/ECLWatchSrc.xml]: queue? it removes the file. Look at the other instances of this text in the manual. They should all be the same. Also, the presentation/format is different than the others. Look at Spray Fixed, for example, on page 80

**Review by JamesDeFabia — CHANGES_REQUESTED** [review]
see inline comments

**Review by JamesDeFabia — APPROVED** [review]
ok from my POV

**Comment by miguelvazq** [comment]
Automatically removing the file from where if its in the process of a copy?

**Comment by JamesDeFabia** [comment]
> @miguelvazq  said:
> Automatically removing the file from where if it is in the process of a copy?

Expiring does not remove it from the copy process. The copy process is brief, it should NEVER take days :) 
Setting an expiration date on a file removes it from the Distributed File System (both physical file and logical file definition) upon expiration. The Expires in (Days) setting adds the expiration date during the copy process.

**Comment by miguelvazq** [comment]
Looks good.

**Comment by JamesDeFabia** [comment]
@richardkchapman This is good to merge now

**Pull request merged**